src/user/profile/content/PostContent.js

```diff
@@ -0,0 +1,148 @@
+    const { post } = this.state;
+
+    return (
+      <Modal className="modal" show={show} onHide={onHide} centered>
+        <div className="content-cards-designs">
+          <div className="content-clone-card">
+            <div className="content-card-main-content">
+              <div className="content-card-header">
+                <div className="user-post-info">
+                  <div className="user-thumb">
+                    <img
+                      src="https://randomuser.me/api/portraits/med/men/75.jpg"
+                      className="img-responsive"
+                    />
+                  </div>
+                  <div className="user-information">
+                    <p>{`${post.userId?.name.firstName} ${post.userId?.name.lastName}`}</p>
+                    <small>{post.createdAt}</small>
+                  </div>
+                </div>
+                <div className="post-action">
+                  <i className="fas fa-ellipsis-h"></i>
+                </div>
+              </div>
+
+              <div className="content-card-body simple-text-card">
+                <p>{post.content}</p>
+              </div>
+            </div>
+
+            <div className="content-card-like-comment-holder">
+              <div className="content-card-like-comment"></div>
+            </div>
+
+            <div className="content-card-actions-holder">
+              <div className="content-card-actions">
+                <div className="content-btn-holder">
+                  <a href="#">
+                    <i className="far fa-thumbs-up"></i> Like
+                  </a>
+                </div>
+                <div
+                  className="content-btn-holder"
+                  onClick={this.handleComments}
+                >
+                  <a href="#">
+                    <i className="far fa-comment-alt"></i> Comment
+                  </a>
+                </div>
+              </div>
+            </div>
+
+            <div className="content-card-comments">
+              <div className="comment-input-holder">
+                <div className="user-thumb">
+                  <img
+                    src="https://randomuser.me/api/portraits/med/men/75.jpg"
+                    className="img-responsive"
+                  />
+                </div>
+                <div className="comment-input">
+                  <div
+                    className="comment-box"
+                    placeholder="Write a comment..."
+                    contentEditable="true"
+                  ></div>
+                </div>
+              </div>
+              <div className="comment-container">
+                {this.state.comments?.map((commentItem, index) => {
+                  return (
+                    <div className="comment-item" key={index}>
+                      <div className="user-icon">
+                        <img
+                          src="https://randomuser.me/api/portraits/med/men/75.jpg"
```

Comment:
@AuraOfDivinity Please use the user image that would come from backend, or as of now use any placeholder instead of external API call.